### PR TITLE
Removes duplicate code from handle_status_effects

### LINKED ISSUE #15916: dizziness, confused, drowsyness,  and jitteriness are being lowered two times per life tick for human mobs
**Issue Description**:
Dizziness is being reduced in two separate places for human mobs. Once in `/mob/living/carbon/handle_status_effects()` and once in `/mob/living/carbon/human/handle_chemicals_in_body()`
Resulting in a reduction of 4 dizziness per tick if you're not resting and a reduction of 20 if you are resting.

`/mob/living/proc/Life(seconds, times_fired)` is the one calling both calls

The same goes for the other effects

**What did you expect to happen**: 
It only to reduced once.

**What happened instead**:
It got reduced multiple times. Causing weird behaviour.

**Why is this bad/What are the consequences**:
It should only really be done once as this creates odd behaviour for this status effect

**Steps to reproduce the problem**:
Get dizzy

**When did the problem start happening**:
Unsure. Might be caused by the life refactor a while back.

**Extra information**:
My expectation is that `handle_chemicals_in_body` should not handle this. A maint has to confirm this though @Fox-McCloud

### PR DESCRIPTION
**Pull request by Dragonkiller93**
## What Does This PR Do
Fixes #15916, removes duplicate code for dizziness, drowsiness, and jitteriness lowering from the handle_status_effects function. The same functionality seems to be already present in carbon/human/life/handle_chemicals_in_body.

## Why It's Good For The Game
Hopefully stops these status effects going down faster than they should.

## Changelog
:cl:Iwanabeu
fix: Drowsiness, Dizziness, Confusion, and Jitteriness time is decreased properly.
/:cl:

### CODE REVIEW

**Comment by Dragonkiller93** [comment]
Could someone help me with these tests? I have no idea what they mean

**Review by DeveloperLoser — COMMENTED** [review]
I may be wrong because i'm pretty new to this but it seems like you just want to delete this to fix your error.

`code/modules/mob/living/carbon/life.dm:266:warning: restingpwr: variable defined but not used`

Can't do any testing myself but definitely do some things on your local server and make sure it doesn't blow up.

**Comment by Dragonkiller93** [comment]
Okay, I followed mochi's advice and removed the code from carbon/human, I tested it on drowsiness and jitteriness and both seem to function correctly, not sure if there was a reason for the code in handle_chemicals_in_body(). If there was, this PR probably isn't needed.

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -314,6 +314,12 @@
 		do_jitter_animation(jitteriness)
 		AdjustJitter(-restingpwr)
 
+
+
+	//Jitteryness
+	if(jitteriness)
+		do_jitter_animation(jitteriness)
+
```
Comment (dearmochi) [inline code/modules/mob/living/carbon/life.dm:322]:
```suggestion
```

Unneeded. It's being done right above.

**Review by dearmochi — CHANGES_REQUESTED** [review]

**Review by dearmochi — APPROVED** [review]

**Pull request merged**